eessi_bot_job_manager.py

```diff
@@ -667,14 +674,26 @@ def main():
     if max_iter != 0:
         known_jobs = job_manager.get_known_jobs()
     while max_iter < 0 or i < max_iter:
+        # sleep poll_interval seconds (not for the first iteration)
+        if i != 0 and (max_iter < 0 or i < max_iter):
+            log(
+                "job manager main loop: sleep %d seconds" % poll_interval,
+                job_manager.logfile,
+            )
+            time.sleep(poll_interval)
         log("job manager main loop: iteration %d" % i, job_manager.logfile)
         log(
             "job manager main loop: known_jobs='%s'" % ",".join(
                 known_jobs.keys()),
             job_manager.logfile,
         )
 
-        current_jobs = job_manager.get_current_jobs()
+        try:
+            current_jobs = job_manager.get_current_jobs()
+        except:
```

Comment:
I think, `run_cmd` raises a `RuntimeError`, so the line could be changed as follows
```suggestion
        except RuntimeError:
```